### DISCUSSION TITLE
Notes field for attachments

### DISCUSSION THREAD

**Discussion by slavaf2000** [keepassxreboot/keepassxc#9639] [Ideas]
It would be useful if I could write notes for each attachment which appears in the "Advanced" section of an entry. 

Currently, there are two fields there: Name and Size of the attached file. I sometimes have multiples files for different purposes, and it would be useful if I could also record a note beside each file.

Would appreciate if this small feature is implemented sometime.

Many thanks!

**Comment by droidmonkey** [comment]
You can use the actual entry notes or add advanced attributes. KeePass file format has no support for storing anything but file name for attachments. 

You can also just name your attachments to be more descriptive of their purpose.

**Reply by slavaf2000** [reply]
Sounds good, thank you for your response. Thought I would ask! I am not familiar with the internal structure.
